pykickstart/commands/snapshot.py

```diff
@@ -110,17 +113,9 @@ def _getParser(self):
     def parse(self, args):
         (ns, extra) = self.op.parse_known_args(args=args, lineno=self.lineno)
```

Comment:
You can call `parse_args` now.

It will probably fix the pylint warning:
`W0612(unused-variable):114,13: F26_Snapshot.parse: Unused variable 'extra'`

Reply:
Fixed.